app/controllers/api/projects.php

```diff
@@ -111,35 +111,8 @@
 
         $projectId = ($projectId == 'unique()') ? ID::unique() : $projectId;
 
-        $backups['database_db_fra1_v14x_02'] = ['from' => '03:00', 'to' => '05:00'];
-        $backups['database_db_fra1_v14x_03'] = ['from' => '00:00', 'to' => '02:00'];
-        $backups['database_db_fra1_v14x_04'] = ['from' => '00:00', 'to' => '02:00'];
-        $backups['database_db_fra1_v14x_05'] = ['from' => '00:00', 'to' => '02:00'];
-        $backups['database_db_fra1_v14x_06'] = ['from' => '00:00', 'to' => '02:00'];
-        $backups['database_db_fra1_v14x_07'] = ['from' => '00:00', 'to' => '02:00'];
```

Comment:
Why are we removing these?

Reply:
We don't have these DBs anymore, or any managed DBs, so I think it's redundant. cc @christyjacob4 to confirm

Reply:
Yeap agreed. We needed this code to prevent DDL operations being performed on a DB undergoing backup. Its not required anymore